cmds/ls/fileinfo_freebsd.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"regexp"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	humanize "github.com/dustin/go-humanize"
+)
+
+// From Linux header: /include/uapi/linux/kdev_t.h
```

Comment:
is that correct? linux header info in freebsd?

Reply:
Fixed.